lib/swaps/Swaps.ts

```diff
@@ -130,23 +133,14 @@ class Swaps extends EventEmitter {
    * @returns undefined if the setup is verified, otherwise an error message
    */
   private verifyLndSetup = (deal: SwapDeal, peer: Peer) => {
+    // TODO: this verification should happen before we accept orders from the peer
```

Comment:
This is a good point and I'll probably take care of this separately. I'll see about updating #822 to include this logic or just save it for another PR if it looks like it would be too complicated.